x/marketmap/keeper/keeper.go

```diff
@@ -89,7 +89,18 @@ func (k *Keeper) GetMarket(ctx context.Context, tickerStr string) (types.Market,
 
 // setMarket sets a market.
 func (k *Keeper) setMarket(ctx context.Context, market types.Market) error {
-	return k.markets.Set(ctx, types.TickerString(market.Ticker.String()), market)
+	if err := k.markets.Set(ctx, types.TickerString(market.Ticker.String()), market); err != nil {
```

Comment:
Just curious, what are the properties of this cosmos-sdk collections map? Does it handle stuff to ensure things are deterministic? Any idea of the specifics?